src/main/java/org/folio/persist/InstanceMarcRepository.java

```diff
@@ -0,0 +1,15 @@
+package org.folio.persist;
+
+import static org.folio.rest.persist.PgUtil.postgresClient;
+
+import io.vertx.core.Context;
+import java.util.Map;
+import org.folio.rest.jaxrs.model.MarcJson;
+
+public class InstanceMarcRepository extends AbstractRepository<MarcJson> {
+  public static final String INSTANCE_SOURCE_MARC_TABLE = "instance_source_marc";
+
+  public InstanceMarcRepository(Context context, Map<String, String> okapiHeaders) {
+    super(postgresClient(context, okapiHeaders), INSTANCE_SOURCE_MARC_TABLE, MarcJson.class);
+  }
+}
```

Comment:
What is this used for? Was the intention that this would provide the MARC record associated with the instance?

Reply:
Right now it is used only to remove all records in the table when an instance/all instances removed. It relies on RMB methods instead of using a self written delete SQL query.